src/App.js

```diff
@@ -1,13 +1,55 @@
 import React from 'react';
+import { GoodsList } from './components/GoodsList/GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  renderGoods = (method) => {
+    method.then(goods => this.setState({
+      goods,
+    }));
+  }
```

Comment:
```suggestion
  renderGoods = () => {
    getAll()
    .then(goods => this.setState({
      goods,
    }));
  }
```
Мне кажется логичнее, чтобы вызов запроса был тут